pkg/odo/util/cmdutils.go

```diff
@@ -8,14 +8,13 @@ import (
 	"github.com/redhat-developer/odo/pkg/log"
 	"github.com/redhat-developer/odo/pkg/machineoutput"
 
-	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // LogErrorAndExit prints the cause of the given error and exits the code with an
 // exit code of 1.
 // If the context is provided, then that is printed, if not, then the cause is
-// detected using errors.Cause(err)
+// displayed
```

Comment:
The documentation of this method could be more accurate, I think, by stating that the given error is displayed in all cases, not just its cause. Something like this, maybe:
```go
// LogErrorAndExit prints the given error and exits the code with an exit code of 1.
// If the context is provided, then that is printed alongside the error.
```